samples/simple-chat/csharp/FunctionApp/Functions.cs

```diff
@@ -49,10 +47,26 @@ public static class Functions
         //    return connectionInfo;
         //}
 
+        //Boardcast to multiple endpoints with routing logic. Only works in persistent mode.Transient mode always boardcast to the default service endpoint and ignore routing logic.
+        [FunctionName("selectiveBroadcast")]
```

Comment:
how about move the multi endpoint sample to another folder, like other samples? It's better to keep the simple-chat one simple enough